docs/FAQ.rst

```diff
@@ -167,6 +167,16 @@ LightGBM
 
 --------------
 
+-  **Question 12**: Why is early stopping not enabled by default in LightGBM?
+
+-  **Solution 12**: Early stopping involves choosing a validation set, a special type of holdout which is used to evaluate the current state of the model after each iteration to see if training can stop.
+
+    In ``LightGBM``, `we have decided to require that users specify this set directly <./Python-Intro.html#early-stopping>`_. Many options exist for splitting training data into training, test, and validation sets.
```

Comment:
Since this FAQ entry is in general section, not in `Python` one, link should point to something language package independent, I think. Maybe use this link https://github.com/microsoft/LightGBM/blob/master/docs/Parameters.rst#valid?

Reply:
yep good point, I'll revise. Thank you!

GitHub won't let me comment on your note about the indentation so I'll tell you here...I'll fix that too and actually run `make html` and do a visual inspection.

Reply:
![image](https://user-images.githubusercontent.com/7608904/62000156-f9daea00-b095-11e9-9717-73217460bd77.png)

^ think I've fixed that bad indentation now. Added this link as well.

I also touched one line in `README.rst`, saw an unnecessary line break.

I also rebased to `master` since there have been some changes since I first opened this PR.